artifacts/definitions/Windows/Registry/WDigest.yaml

```diff
@@ -0,0 +1,55 @@
+name: Windows.Registry.WDigest
+description: |
+    Find WDigest registry values on the filesystem.
+
+    In order to prevent the “clear-text” password from being placed in LSASS, the following registry key needs to be set to “0” (Digest Disabled):
+
+     - HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\WDigest “UseLogonCredential”(DWORD)
+
+    This registry key is worth monitoring in an environment as an attacker may wish to set it to 1 to enable Digest password support which forces “clear-text” passwords to be placed in LSASS on any version of Windows from Windows 7 / 2008R2 up to Windows 10 / 2012R2. Furthermore, Windows 8.1 / 2012 R2 and newer do not have a “UseLogonCredential” DWORD value, so the key needs to be added. The existence of the key is suspicious, if not expected.
+
+type: CLIENT
+
+author: Eduardo Mattos - @eduardfir
+
+precondition:
+  SELECT * FROM info() where OS = 'windows'
+
+parameters:
+  - name: tactic
```

Comment:
Same as above: These were from an idea from our friend MattG, to support a future feature of hunting in accordance to MITRE TTPS. Example: IR case where client was hit with Sodin. We know what Sodin's TTPS are and we can create a hunt based on them (e.g.: https://thedfirreport.com/2021/03/29/sodinokibi-aka-revil-ransomware/)
They are not useful at this moment however, it was more of a preparation for the future, so it could be removed